app/modules/info/hiddenprofilecard.py

```diff
@@ -0,0 +1,59 @@
+import discord
+from discord.ext import commands
+from discord.ext.commands import Context
+
+from app.utilities import logger
+import app.utilities.drawing.master as drawing
+
+
+ContextType = commands.Context
+
+
+def profileCardFromMember(member):
+
+    # getting user's server specific nickname (if it's not set, then just getting his general discord name)
+    nickname = member.nick
+    if not nickname:
+        nickname = member.name
```

Comment:
member.display_name will get the member's nickname if it exists, or else it gets the member's username, so you could just have:

```suggestion
    nickname = member.display_name
```